src/prpl.ts

```diff
@@ -132,7 +132,7 @@ function loadBuilds(root: string, config: ProjectConfig|undefined): Build[] {
     // No builds were specified. Try to serve an entrypoint from the root
     // directory, with no capability requirements.
     console.warn(`WARNING: No builds configured.`);
-    builds.push(new Build(0, new Set(), path.join(root, entrypoint)));
+    builds.push(new Build(0, new Set(), entrypoint));
```

Comment:
Yeah, the bug I caught (hooray tests) was that everything is always served relative to the server root by send(), so adding it here is actually wrong. It worked in most of my testing because my root was always ".", so that part of the join was a no-op.